tests/base.py

```diff
@@ -54,7 +55,10 @@ class PyverbsAPITestCase(unittest.TestCase):
     """
     def __init__(self, methodName='runTest', dev_name=None):
         super(PyverbsAPITestCase, self).__init__(methodName)
-        self.dev_name = dev_name
+        if parser.args['dev_name']:
```

Comment:
Why keep the dev_name argument at all?

'parser' should be a static class member of PyverbsAPITestCase, and give parser a better name, like 'config' or something.

Reply:
Fixed